src/core/amount.js

```diff
@@ -21,7 +22,7 @@ function Amount(value = new XRPValue(NaN)) {
   this._value = value;
   this._is_native = true; // Default to XRP. Only valid if value is not NaN.
   this._currency = new Currency();
-  this._issuer = new UInt160();
+  this._issuer = 'NaN';
```

Comment:
Because it's a string field. I just used the string 'NaN' because that's what was required to not break the unit tests.